etc/cvmfs/osgstorage-auth.conf

```diff
@@ -9,26 +9,46 @@ X509_CERT_DIR=
 
 CVMFS_MAX_RETRIES=0
 
-CVMFS_EXTERNAL_URL="https://unl-cache.nationalresearchplatform.org:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://xrootd-local.unl.edu:1094/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stashcache.t2.ucsd.edu:8443/"
+# authenticated OSDF caches used by OSG
```

Comment:
Same here, delete this comment line for this branch.

Reply:
done